dist/arch/PKGBUILD.bin

```diff
@@ -0,0 +1,27 @@
+# Maintainer: tectonic-deploy <sasha+tectonic@hackafe.net>
+# Maintainer: Daniel M. Capella <polyzen@archlinux.org>
+# Contributor: Jan Tojnar <jtojnar@gmail.com>
+
+# The master version of this file is maintained here:
+#
+#   https://github.com/tectonic-typesetting/tectonic/blob/master/dist/arch/PKGBUILD.in
+#
+# The version on aur.archlinux.org is updated automatically by the Tectonic
+# CI/CD system # when new versions are released.
+
+pkgname=tectonic
+pkgver=0.3.3
+pkgrel=0
+arch=('x86_64')
+pkgdesc='Modernized, complete, self-contained TeX/LaTeX engine, powered by XeTeX and TeXLive'
```

Comment:
The description should probably mention that this is the pre-built binary package variant.